website_sale_product_description/models/product_template.py

```diff
@@ -7,7 +7,6 @@
 _logger = logging.getLogger(__name__)
 
 
-# configAuthorization.set('PRISMA f3d8b72c94ab4a06be2ef7c95490f7d3')
 
```

Comment:
we have a pylint error of two many empty lines, please fix this
https://travis-ci.org/ingadhoc/website/jobs/394054614#L828